contracts/farm/FeeExtractAndDistribute_Arbitrum.sol

```diff
@@ -203,4 +205,16 @@ contract FeeExtractAndDistribute_Arbitrum is PausableGuardian_0_8 {
             );
         }
     }
+
+    function setSlippage(uint32 newSlippage) external onlyGuardian {
+        slippage = newSlippage;
```

Comment:
oh I see. but shouldn't slippage be per coin?

Reply:
we are only bridging USDC

Reply:
oh right